README.md

```diff
@@ -1,75 +1,172 @@
-![](https://img.shields.io/github/contributors/open-horizon-services/web-helloworld-python)
 
-Extremely simple HTTP server (written in Python) that responds on port 8000 with a hello message.
+![License](https://img.shields.io/github/license/open-horizon-services/web-helloworld-python)
+![Architecture](https://img.shields.io/badge/architecture-x86,arm64-green)
+![Contributors](https://img.shields.io/github/contributors/open-horizon-services/web-helloworld-python.svg)
 
-Begin by editing the variables at the top of the Makefile as desired. If you plan to push it to a Docker registery, make sure you give your docker ID. You may also want to create unique names for your **service** and **pattern** (necessary if you are sharing a tenancy with other users and you are all publishing this service).
+This Open Horizon service demonstrates a simple HTTP server written in Python. The server responds with a "Hello, World!" message on port 8000. To check the "Hello, World!" message in your web browser, navigate to http://localhost:8000/.
 
-To play with this outside of Open Horizon:
+## Prerequisites
 
-```
+To ensure the successful installation and operation of the Open Horizon service, the following prerequisites must be met:
+
+
+**Open Horizon Management Hub:** To publish this service and register your edge node, you must either [install the Open Horizon Management Hub](https://open-horizon.github.io/quick-start) or have access to an existing hub. You may also choose a downstream commercial distribution like IBM's Edge Application Manager. If you'd like to use the Open Horizon community hub, you may [apply for a temporary account](https://wiki.lfedge.org/display/LE/Open+Horizon+Management+Hub+Developer+Instance) at the Open Horizon community hub, where credentials will be provided.
+
+**Edge Node:**You will need an x86 computer running Linux or macOS, or an ARM64 device such as a Raspberry Pi running Raspberry Pi OS or Ubuntu. The `anax` agent software must be installed on your edge node. This software facilitates communication with the Management Hub and manages the deployment of services.
+
+**Optional Utilities:** Depending on your operating system, you may use:
+  - `brew` on macOS
+  - `apt-get` on Ubuntu or Raspberry Pi OS
+  - `yum` on Fedora
+  
+  These commands can install `gcc`, `make`, `git`, `jq`, `curl`, and `net-tools`. These utilities are not strictly required but are highly recommended for successful deployment and troubleshooting.
+
+
+## Installation
+
+1. **Clone the repository:**
+    Clone the `web-helloworld-python` GitHub repo from a terminal prompt on the edge node and enter the folder where the artifacts were copied.
+
+   ```shell
+   git clone https://github.com/open-horizon-services/web-helloworld-python.git
+   cd web-helloworld-python
+    ```
+
+2. **Edit Makefile:**
+    Adjust the variables at the top of the Makefile as needed, including your Docker ID and unique names for your service and pattern.
```

Comment:
I see you have a mention here, but maybe you could add a little block showing the values with their defaults and if they `export` them in their terminal they don't have to literally edit the values in the `Makefile` to override them